recipes/nas/all/conandata.yml

```diff
@@ -1,6 +1,16 @@
 sources:
+  "1.9.5":
+    - url: "https://downloads.sourceforge.net/nas/nas-1.9.5.tar.gz"
+      sha256: "b7884afb38feec03a196bd3b7e9c47b803c830ecd10d7455e9c97e122c37944c"
+    - url: "https://unlicense.org/UNLICENSE"
+      sha256: "7e12e5df4bae12cb21581ba157ced20e1986a0508dd10d0e8a4ab9a4cf94e85c"
   "1.9.4":
-    - url: "https://downloads.sourceforge.net/nas/nas-1.9.4.src.tar.gz"
+    - url: "https://downloads.sourceforge.net/project/nas/nas/nas%201.9.4%20%28stable%29/nas-1.9.4.src.tar.gz"
       sha256: "cf36ea63751ce86cfd3b76c1659ce0d6a361a2e7cb34069854e156532703b39d"
     - url: "https://unlicense.org/UNLICENSE"
       sha256: "7e12e5df4bae12cb21581ba157ced20e1986a0508dd10d0e8a4ab9a4cf94e85c"
+patches:
+  "1.9.4":
+    - patch_file: "patches/0001-1.9.4-extern-variable-bugfix.patch"
+      patch_description: "Extern variable to fix bug"
+      patch_type: "bugfix"
```

Comment:
bugfix are for official things from upstream, if this is a compiler/platform issue should be portability :)